core/messaging/__init__.py

```diff
@@ -4,15 +4,15 @@
 class Message:
     """Generic Message class that defines the components of a validation error message"""
 
-    def __init__(self, sheet, section, cell_index, description, error_type):
+    def __init__(self, sheet, section, cell_indexes: tuple[str, ...] | None, description, error_type):
         self.sheet = sheet
         self.section = section
-        self.cell_index = cell_index
+        self.__cell_indexes: None | tuple[str] = cell_indexes
```

Comment:
Type hint here is `None | tuple[str]` while type hint for argument `cell_indexes` is `tuple[str, ...] | None`, can we make consistent?

Reply:
Good spot 👍  Flagged this with Gids last week - we defo need to add something that will actually enforce the type hints otherwise it's fairly pointless. I'll make a ticket for that and maybe just look at doing it ...